tests/integration/bond_test.py

```diff
@@ -369,3 +376,30 @@ def test_bond_with_empty_ipv6_static_address(eth1_up):
         assertlib.assert_state(bond_state)
 
     assertlib.assert_absent('bond99')
+
+
+@pytest.fixture
+def linux_br0_over_vlan101(eth1_up):
```

Comment:
The name states you define a bridge over a vlan in this fixture, but in fact, in the body you define a bridge over a regular interface (eth1). This is not the scenario that the PR is all about.